app/assets/stylesheets/custom.scss

```diff
@@ -176,3 +176,17 @@ input {
     color: $state-danger-text;
   }
 }
+
+.checkbox {
+  margin-top: -10px;
+  margin-bottom: 10px;
+  span {
+    margin-left: 20px;
+    font-weight: normal;
+  }
+}
+
+#session_remember_me {
```

Comment:
can you change this to `class` ? I don't think `id` is the good choice.

Reply:
Can you explain to me why id is a good choice?

Reply:
`class` is better, because It can be shared among many elements, while `id` can only be used for one element

Reply:
Ohh, I'm sorry, I misread your comment. I understand, thank you so much.